Fix `key` and styles warning

FiberJW opened the pull request:
This fixes the need to index list-rendered components and allow people to use style objects instead of stylesheets working here: https://github.com/fiber-god/one-punch-fitness/blob/develop/src/components/Ticker.js https://github.com/fiber-god/one-punch-fitness/blob/develop/src/screens/Workout/components/Timer/Timer.re

FiberJW committed:
Fix `key` and styles warning

This fixes the need to index list-rendered components and allow people to use style objects instead of stylesheets

browniefed commented:
Hey. I had already fixed the key warning. Thanks for the style prop fix.
Can you rebase please? I added in custom duration after you forked.

Thanks

FiberJW commented:
I'm AFK right now. If you're still working on it right now you can just
push the proposed changes yourself and close the PR

On Sat, Jan 20, 2018, 1:20 PM Jason Brown <notifications@github.com> wrote:

> Hey. I had already fixed the key warning. Thanks for the style prop fix.
> Can you rebase please? I added in custom duration after you forked.
>
> Thanks
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/browniefed/react-native-ticker/pull/1#issuecomment-359191387>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AL6Qelzvp52nwnrS4vdo2SQWMet_88-Vks5tMi53gaJpZM4Rlh2d>
> .
>
